package.json

```diff
@@ -10,29 +10,33 @@
   "main": "lib/index.js",
   "author": "Eric Ferraiuolo <edf@ericf.me>",
   "dependencies": {
-    "babel-runtime": "^6.2.0",
+    "@babel/runtime": "7.0.0",
```

Comment:
Why it's not semver style like `^7.0.0`? any concern?

Reply:
fixed. thanks

didn't notice. i just ran babel-upgrade https://github.com/babel/babel-upgrade